pages/documentation/simulation/powerflow/openlf.md

```diff
@@ -148,10 +148,7 @@ If `balanceType` equals to `PROPORTIONAL_TO_CONFORM_LOAD`, the power factor rema
 The default value for `loadPowerFactorConstant` property is `false`.
 
 **dcUseTransformerRatio**  
-The `dcUseTransformerRatio` property is an optional property that defines if ratio of transformers should be used in the flow equations during DC approximation. The default value of this parameter is `false`.
-
-**updateFlows**  
```

Comment:
If this property is deprecated or removed, keep it in a `Deprecated properties` or `Removed properties` section where you indicate since which version it has been deprecated or removed

Reply:
This parameter was a mistake indeed.